CHANGELOG.md

```diff
@@ -4,10 +4,18 @@
 
 ### Added
 
+  - misc: add possibility to do custom builds through environment variables, to remove unwanted features from minified code.
+  - languages/dash: allow plain text TTML, SAMI, SRT and VTT subtitles in DASH and Smooth manifests
+  - languages: add possibility to show fragmented VTT text tracks in <track> element, even for browser that do not support webvtt
+  - languages: add possibility to show TTML, SRT, VTT and SAMI text tracks in HTML tags for richer stylisation.
+  - languages: add SRT texttrack support in <track> elements ("native" textTrackMode)
+  - api: add ``textTrackElement`` option to ``loadVideo``
```

Comment:
"textTrackElement" is added to the constructor, isn't it ?

Reply:
Those commits are here because this branch is based on https://github.com/canalplus/rx-player/pull/72.

The ``textTrackElement`` option is added only to ``loadVideo``, as adding it to both the constructor and loadVideo would mostly complexify our API.